### PR TITLE
[IMP] project_task_pull_request: PR on task

### PR DESCRIPTION
**Pull request by remytms**
PR de @nicolasj77.

Migrate  project_task_pull_request from 10.0 to 9.0

### CODE REVIEW

**Comment by nicolasj77** [comment]
@remytms 
Je peux merger ?

File: project_task_pull_request/models/project_task.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Specialty Medical Drugstore
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import _, api, exceptions, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+    _name = "project.task"
```
Comment (houssine78) [inline project_task_pull_request/models/project_task.py]: you can remove _name when inheriting

File: project_task_pull_request/models/project_project.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Specialty Medical Drugstore
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import fields, models
+
+
+class Project(models.Model):
+    _inherit = "project.project"
+    _name = "project.project"
```
Comment (houssine78) [inline project_task_pull_request/models/project_project.py]: same here

File: project_task_pull_request/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Project Task Pull Request",
+    "summary": "Adds a field for a PR URI to project tasks",
+    "version": "9.0.1.0.0",
+    "category": "Project Management",
+    "website": "https://odoo-community.org/",
+    "author": "SMDrugstore, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
```
Comment (houssine78) [inline project_task_pull_request/__openerp__.py]: can be removed if False as it's the default value

**Review by houssine78 — APPROVED** [review]
can be merged after take into account my minor feedback

File: project_task_pull_request/models/project_task.py (hunk 1 above)
Comment (nicolasj77) [reply project_task_pull_request/models/project_task.py]: Done

File: project_task_pull_request/models/project_project.py (hunk 2 above)
Comment (nicolasj77) [reply project_task_pull_request/models/project_project.py]: Done

File: project_task_pull_request/__openerp__.py (hunk 3 above)
Comment (nicolasj77) [reply project_task_pull_request/__openerp__.py]: Done

**Pull request merged**